recipe/meta.yaml

```diff
@@ -170,6 +170,11 @@ requirements:
     - {{ pin_compatible("nss") }}        # [unix]
     - {{ pin_compatible("nspr") }}       # [unix]
     - libxcb                             # [linux]
+    - xcb-util                           # [linux]
```

Comment:
I feel like these should be added to `requirements` and `run_exports` should be used in the `run` section. Otherwise you are really linking to the system `xcb` stuff indirectly aren't you?

Reply:
I'm not sure that I understand.  First of all, these libraries are run requirements, not build requirements.  (In order to build we need the `{{ cdt('xcb-util-*-devel') }}` libraries which are listed in the build section).  But there may be something about the way `run_exports` works that I don't understand.

Second, when these libraries are installed by conda into an env they will be used instead of any system libraries that may also be installed because of the way the link order works.